### ISSUE TITLE
Range label for 'Layer\Filter\Decimals' is using priceCurrency->format

### ISSUE THREAD

**Issue by roobware** [magento/magento2#16453]
The decimal filter is using the currency formatter.
Is this so planed or a bug?

This two files are affected:
vendor/magento/module-catalog/Model/Layer/Filter/Decimal.php `->_renderItemLabel()`
vendor/magento/module-catalog-search/Model/Layer/Filter/Decimal.php `-> renderRangeLabel()`

![catalog_filter_decimal](https://user-images.githubusercontent.com/27640321/42057362-6325c7d2-7b1d-11e8-8421-6fa2a1e5cafd.png)

![catalogsearch_filter_decimal](https://user-images.githubusercontent.com/27640321/42057364-635e2014-7b1d-11e8-8b7f-b3bfbec4a3fa.png)

I think these filters should not use the currency formatter and it should be possible to define the unit of value in the eav table for the label :)

### Preconditions
Debian 9.4
Apache 2.4
PHP 7.1
MySQL 5.7
Magento 2.2.5

### Steps to reproduce
Default standard installation.
Add a new eav attribute programmatically with data type 'decimal'.

### Expected result
Price filter should generate range labels with numbers only:
- 0 - 9
- 10 - 19
- 20 and above.......

### Actual result
- 0,00 € - 9,00 €
- 10,00 € - 9,00 €
- 20,00 € - -0,01 €

**Comment by magento-engcom-team** [comment]
Hi @xBitsSec. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me {$VERSION} instance` 

where `{$VERSION}` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (2.2-develop +). 
For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation.

@xBitsSec do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by ghost** [comment]
@xBitsSec, thank you for your report.
We've acknowledged the issue and added to our backlog.

**Comment by ci2014** [comment]
This issue is still valid for 2.3.3

**Comment by drew7721** [comment]
@ci2014 Yes, this is still valid and it gets worse in 2.3.4 as decimal attributes are actually assigned the price values.

**Comment by drew7721** [comment]
@ci2014 Yes, this is still valid and it gets worse in 2.3.4 as decimal attributes are actually assigned the price values. I will open another issue for this though.

**Comment by ci2014** [comment]
still an issue on Magento ver. 2.3.6.
